sim/src/lib.rs

```diff
@@ -19,10 +19,15 @@ use map_model::{
     Path, PathConstraints, PathRequest, Position,
 };
 
+pub use crate::render::{
```

Comment:
Hmm, I don't really understand the rule for breaking this out into a separate block, but OK.

Reply:
The others are `self::` whereas this is `crate::`. Not sure why though